backboard-example/build.gradle

```diff
@@ -1,4 +1,7 @@
-apply plugin: 'com.android.application'
+plugins {
+    id "com.android.application"
+    id "checkstyle"
```

Comment:
I wonder if we really need checkstyle for the sample app

Reply:
Thank you for pointing this out. I try to preserve the current setup when I move things over to plugin DSL. This was a mistake as I thought `checkstyle` was already part of the setup for the example app. Fixed in d524e1517f26257b81e4e1c977f0070ee6244caf.